sdk/agrifood/azure-verticals-agrifood-farming/README.md

```diff
@@ -32,26 +32,12 @@ Use FarmBeats client library for Python to do the following.
 
 ### Authenticate the client
 
-#### Using Azure Active Directory
+#### Using Microsoft Entra ID
 
-In order to interact with the Azure FarmBeats service, your client must present an Azure Active Directory bearer token to the service.
+In order to interact with the Azure FarmBeats service, your client must present a Microsoft Entra bearer token to the service.
 
-The simplest way of providing a bearer token is to use the `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+The best way of providing a bearer token is to use the Azure Identity library. See the [installation instructions](../../../sdk/identity/azure-identity/README.md#include-the-package) to include `azure-identity`. In the examples, `credential` refers to a `TokenCredential`. It's recommended to use a [credential chain](../../../sdk/identity/azure-identity/readme#construct-a-credential-chain).
```

Comment:
Is it worth just extending the final sentence to explain why a credential chain is recommended? "... to allow for multiple credentials to be specified, and for each credential to fall-through based on the order given." or something - I'm not an expert, but I feel like as a noob I'm left asking "why is it recommended?" I'm sure it is mentioned in the link, but perhaps it is worth spelling it out to be clear right here?